### ISSUE TITLE
Allow stopping progressive rendering when a certain level of convergence has been reached

### ISSUE THREAD

**Issue by dictoon** [appleseedhq/appleseed#1199]

**Comment by Aakash1312** [comment]
How do you define convergence?

**Comment by dictoon** [comment]
Yes, we use the terms "progressive rendering" and "interactive rendering" interchangeably.

Progressive rendering can be stopped manually, or after a given number of samples have been computed.

We would like to add the possibility to automatically stop progressive rendering when a certain level of convergence has been reached. How to define and compute convergence is part of the task.
